okteto.yml

```diff
@@ -1,12 +1,12 @@
 build:
-  todo-list:
-    image: okteto.dev/todo-list
-  
+  go-todo-list:
+    image: okteto.dev/go-todo-list:001
+
 deploy:
-  - cd manifest; kustomize edit set image okteto.dev/todo-list=${OKTETO_BUILD_TODO_LIST_IMAGE}
```

Comment:
The original sample is with `kustomize`, we should keep that.

Reply:
kustomize was not found in those image, that was why i refactored to pure k8s